extension/style.css

```diff
@@ -1,14 +1,40 @@
 .zenhub-status {
   margin: 0.5rem;
   border-radius: 999px;
-  background-color: rgb(246, 248, 250);
-  border-color: rgb(208, 215, 222);
+  background-color: var(--background-color-light);
+  border-color: var(--border-color-light);
   font-family: inherit;
   font-size: 12px;
   height: 20px;
   line-height: 20px;
   font-weight: 600;
-  color: rgb(101, 109, 118);
+  color: var(--text-color-light);
   padding-left: 8px;
   padding-right: 8px;
 }
+
+:root {
+  --text-color-light: rgb(101, 109, 118);
+  --background-color-light: rgb(246, 248, 250);
+  --border-color-light: rgb(208, 215, 222);
+
+  --text-color-dark: rgb(125, 133, 144);
+  --background-color-dark: rgb(22, 27, 34);
+  --border-color-dark: rgb(48, 54, 61);
+}
+
+@media (prefers-color-scheme: light) {
+  .zenhub-status {
```

Comment:
Love this, but afraid I've renamed this to `matthub-select` so a tiny conflict.

Reply:
Conflict fixed.
:thinking:  Now I'm thinking that it can support all the themes in GitHub by:
```
export function getExistingElementClassName(selector: string) {
  const existingSelectElement = document.querySelector(selector);
  return existingSelectElement?.className
}


#setSelectCell
return `<select class="${getExistingElementClassName('[data-testid="single-select-token"]')} matthub-select">${options}</select>`;
```

The CSS after the changes above will be:
```
.matthub-select {
  margin: 0.5rem;
}

.matthub-status-bar {
  padding-top: 0px !important;
}
```